addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1584,6 +1584,14 @@ test("warning on send with shortcut when attempting to post message with still-u
     await contains(".o_notification:has(.o_notification_bar.bg-warning)", {
         text: "Please wait while the file is uploading.",
     });
+    await insertText(".o-mail-Composer-input", "Dummy Message");
```

Comment:
I think we could insert the message before the previous check. I don't think it adds much value to test it without the message and the test will be shorter.